etcdmgmt/etcd-mgmt.go

```diff
@@ -160,3 +147,115 @@ func isETCDStartNeeded() (bool, int) {
 	}
 	return start, pid
 }
+
+func EtcdStartInit() (*os.Process, error) {
```

Comment:
SpawnETCD can be use in place of "StartETCD" function. In this function we are not starting/spawning etcd.